src/test/java/com/thomashan/coup/PlayerCardTest.java

```diff
@@ -0,0 +1,64 @@
+package com.thomashan.coup;
+
+import org.junit.jupiter.api.Test;
+
+import static com.thomashan.coup.Card.AMBASSADOR;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotSame;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class PlayerCardTest {
+    @Test
+    public void testOf_ReturnsPlayerCard() {
+        assertEquals(PlayerCard.class, PlayerCard.of(AMBASSADOR).getClass());
+    }
+
+    @Test
+    public void testReveal_ReturnsANewInstanceOfPlayerCard() {
+        assertNotSame(PlayerCard.of(AMBASSADOR), PlayerCard.of(AMBASSADOR).reveal());
+    }
+
+    @Test
+    public void testReveal_GivenItIsRevealed_ThrowsAnException() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unit tests should not contain more than 1 assert(s).](https://app.codacy.com/app/thomashan/coup-java/pullRequest?prid=4561385)